pulp_ansible/app/viewsets.py

```diff
@@ -373,12 +740,51 @@ def sign(self, request, pk):
         return OperationPostponedResponse(result, request)
 
 
-class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet):
+class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet, RolesMixin):
     """
     AnsibleRepositoryVersion represents a single file repository version.
     """
 
     parent_viewset = AnsibleRepositoryViewSet
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
```

Comment:
RepositoryVersionViewSet doesn't need a `queryset_filtering_required_permission`. I would follow the example in pulp_file for this one: https://github.com/pulp/pulp_file/blob/main/pulp_file/app/viewsets.py#L242